layouts/partials/header.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html>
-    <head>
+    <head {{ if .Site.Params.opengraph }} prefix="og: http://ogp.me/ns#" {{ end }}>
```

Comment:
Please move this to `<html>` tag as per http://ogp.me/, also the spaces inside the if are redundant:

`<html{{ if .Site.Params.opengraph }} prefix="og: http://ogp.me/ns#"{{ end }}>`